Sources/App/CRP/ChangelogSection.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import Stevenson
+
+struct ChangelogSection {
```

Comment:
This is just moved (cut/paste) from [below](https://github.com/babylonhealth/Stevenson/pull/47/files#diff-5cc12f66898eff5595c3de249bbe1851L74) to extract the type in a dedicated file (especially since it's now used by 2 different files)